File: exercises-cpp/2. Condicionales/39.cpp
 #include <iostream>
#include <cstdio>
#include <strings.h>
#include <conio.h>
using namespace std;

int main(){
    system("cls");

    int horas,torniDefec,torniNoDefec,condicion1,condicion2,condicion3;
    string nombreMes;

    printf("Horas de Ausencia: ");cin>>horas;
    printf("Tornillos Defectuosos: ");cin>>torniDefec;
    printf("Tornillos No Defectuosos: ");cin>>torniNoDefec;

    condicion1 = horas < 1.5;
    condicion2 = torniDefec < 300;
    condicion3 = torniNoDefec > 10000;

    if (condicion1 == false and condicion2 == false and condicion3 == false)printf("Grado 5");
    else if (condicion1 == true and condicion2 == false and condicion3 == false)printf("Grado 7");
    else if (condicion1 == false and condicion2 == true and condicion3 == false)printf("Grado 8");
    else if (condicion1 == false and condicion2 == false and condicion3 == true)printf("Grado 9");
    else if (condicion1 == true and condicion2 == true and condicion3 == false)printf("Grado 12");
    else if (condicion1 == true and condicion2 == false and condicion3 == true)printf("Grado 13");
    else if (condicion1 == false and condicion2 == true and condicion3 == true)printf("Grado 15");
    else if (condicion1 == true and condicion2 == true and condicion3 == true)printf("Grado 20");
    else printf("Ningun Grado.");
}

File: exercises-python/2. Problemas de estructura condicional/39.py
'''
39.Desarrolle el programa para obtener el grado de eficiencia de un
operario de torno para obtener el grado de eficiencia de un operario
de torno de una fábrica productora de tornillos de acuerdo a las
siguientes condiciones: No más de 1.5 horas de ausencia al trabajo.
Menos de 300 tornillos defectuosos producidos. Más de 10000 tornillos
no defectuosos producidos. Los grados de eficiencia para cada trabajador
son asignados de la siguiente manera: Si no cumple ninguna condición,
grado 5. Si sólo cumple la primera, grado 7. Si sólo cumple la segunda,
grado 8. Si sólo cumple la tercera, grado 9. Si cumple la primera y segunda
condición, grado 12. Si cumple la primera y tercera condición, grado 13.
Si cumple la segunda y tercera condición, grado 15.
Si cumple las tres condiciones, grado 20.
'''
import os
os.system("cls")

horas = float(input("Horas de Ausencia: "))
torniDefec = int(input("Tornillos Defectuosos:"))
torniNoDefec = int(input("Tornillos No Defectuosos:"))

condicion1 = horas < 1.5
condicion2 = torniDefec < 300
condicion3 = torniNoDefec > 10000

if condicion1 == False and condicion2 == False and condicion3 == False:print("Grado 5")
elif condicion1 == True and condicion2 == False and condicion3 == False:print("Grado 7")
elif condicion1 == False and condicion2 == True and condicion3 == False:print("Grado 8")
elif condicion1 == False and condicion2 == False and condicion3 == True:print("Grado 9")
elif condicion1 == True and condicion2 == True and condicion3 == False:print("Grado 12")
elif condicion1 == True and condicion2 == False and condicion3 == True:print("Grado 13")
elif condicion1 == False and condicion2 == True and condicion3 == True:print("Grado 15")
elif condicion1 == True and condicion2 == True and condicion3 == True:print("Grado 20")
else: print("Ningun Grado.")